src/features/completions.zig

```diff
@@ -136,10 +136,10 @@ fn nodeToCompletion(
     node_handle: Analyser.NodeWithHandle,
     orig_handle: *DocumentStore.Handle,
     orig_name: ?[]const u8,
-    orig_doc: ?[]const u8,
     is_type_val: bool,
     parent_is_type_val: ?bool,
     either_descriptor: ?[]const u8,
+    orig_docs: ?std.ArrayListUnmanaged([]const u8),
```

Comment:
the type of `orig_docs` should be `?[]const []const u8` or `?*std.ArrayListUnmanaged([]const u8)`. passing an immutable `ArrayList` can easily produce memory violations.